script/resolve-ciphersuite.rb

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env ruby -W
```

Comment:
Can we please avoid adding another scripting language dependency to libgit2? We already have a dependency on Python due to clar, so that's the one we should aim for. Just let me know if you feel uncomfortable with rewriting this script in Python and I'll do it for you

Reply:
Well, it's just an one-off script : you only need to use it in case we want to regenerate the mapping list & default ciphers, so the dependency is more transient than clar's Python.

If that's really a problem I can rewrite it, but as I don't expect it to run as part of the build system I didn't think it would matter.